app/src/main/java/org/luxoft/sdl_core/MainActivity.java

```diff
@@ -100,6 +94,50 @@ public void onClick(View view) {
             }
         });
 
+        start_bt_button.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                if (!isBleSupported() || !isBluetoothPermissionGranted()) {
+                    start_bt_button.setEnabled(false);
+                }
+
+                String buttonText = start_bt_button.getText().toString();
+                if(buttonText.equals("Start BT")) {
```

Comment:
@ychernysheva please move start/stop strings to a separate variable in XML resource file with another strings